content/docs/command-reference/exp/push.md

```diff
@@ -6,13 +6,13 @@ to a `dvc remote`.
 ## Synopsis
 
 ```usage
-usage: dvc exp push [-h] [-q | -v] [-f] [--no-cache]
-                    [-r <name>] [-j <number>] [--run-cache]
+usage: dvc exp push [-h] [-q | -v] [-A] [--rev <commit>] [-n <num>] [-f]
+                    [--no-cache] [-r <name>] [-j <number>] [--run-cache]
                     git_remote experiment
 
 positional arguments:
   git_remote          Git remote name or Git URL
-  experiment          Experiment to push
+  experiment          Experiments to push
```

Comment:
Same Q

Reply:
UPDATE: See https://github.com/iterative/dvc/pull/7206#pullrequestreview-924091543

Reply:
Also change line 11 to ` git_remote [experiment [experiment ...]]`?

Reply:
Idk, can we discuss that in a separate PR? Ideally in the core PR related to this. Thanks